root/etc/cont-init.d/50-config

```diff
@@ -8,40 +8,63 @@ mkdir -p \
 	/config/www/assets
 
 # check for .env and copy default if needed
-[[ ! -f "/config/www/config.inc.php" ]] && \
+[[ ! -f "/config/www/config.inc.php" ]] &&
 	cp /var/www/html/config.inc.php-sample /config/www/config.inc.php
 
 # check for zero-length .env and alert user if found
-[[ ! -s "/config/www/config.inc.php" ]] && \
-  echo "WARNING: zero-length config.inc.php file detected. Please delete /config/www/config.inc.php and restart the container"
+[[ ! -s "/config/www/config.inc.php" ]] &&
+	echo "WARNING: zero-length config.inc.php file detected. Please delete /config/www/config.inc.php and restart the container"
 
 # link assets
 rm -rf /var/www/html/assets
 ln -s /config/www/assets /var/www/html/assets
 
```

Comment:
I agree, that it is a good idea to prefix the env variables with `MRBS`.
However, it is important for this to be backwards compatible, so please check if the old variables are set and "translate" them to the new ones.